pallets/identity/src/lib.rs

```diff
@@ -627,25 +627,6 @@ decl_module! {
             Self::set_frozen_signing_key_flags(origin, false)
         }
 
-        #[weight = SimpleDispatchInfo::FixedNormal(50_000)]
-        pub fn get_my_did(origin) -> DispatchResult {
-            let sender_key = AccountKey::try_from(ensure_signed(origin)?.encode())?;
-            let did = Context::current_identity_or::<Self>(&sender_key)?;
-
-            Self::deposit_event(RawEvent::DidQuery(sender_key, did));
-            Ok(())
-        }
-
-        #[weight = SimpleDispatchInfo::FixedNormal(200_000)]
-        pub fn get_cdd_of(_origin, of: T::AccountId) -> DispatchResult {
```

Comment:
These are super handy during testing, can we please keep them for the testnet? :)

Reply:
What do you use them for? My thinking is that we should just be able to use an RPC function instead.

Reply:
I'll add them back in though if needed - doesn't matter much either way - was just tidying up ;-)

Reply:
It's easier to call these functions from the UI than to make an RPC call :)